Projects/Foundations/Schemas/Docs-Tutorials/D/Developers/Developers-Intro/developers-intro-020-tutorial-topic-files-component.json

```diff
@@ -33,7 +41,11 @@
             "text": "Market Files: For time-frames of 1 hour and above, it is convenient to have a single file with all the data of the whole market.",
             "translations": [
                 {
-                    "language": "RU",
+                    "language": "DE",
+                    "text": "Market Files: Für Zeitrahmen von 1 Stunde und mehr ist es praktisch, eine einzige Datei mit allen Daten des gesamten Marktes zu haben.",
+                    "updated": 1641736760116
+        },
+        {                    "language": "RU",
```

Comment:
Nearly all files look great now. The only ones left are the some files where: "language": "RU",   is too far right and the brackets are to far left.